### DISCUSSION TITLE
Background image different in each list page

### DISCUSSION THREAD

**Discussion by danraymond** [nunocoracao/blowfish#556] [Ideas]
I am loving theme have to say!

When  a list page is shown e.g. the Documentation Page that shows all the documentation articles, the background is the  site wide one.

Is it possible to somehow include an image in the front matter of the _index.md file to show a different image  behind just like the actual articles do??

**Comment by arielgoes** [comment]
I support this idea. Also it would be cool to allow different background for each article. For now we can only perform this with the "featured" image

**Comment by nunocoracao** [comment]
@danraymond and @arielgoes both are already possible by placing an image called featured.* in the folder where the index file lives
